### PR TITLE
feat: Seed Garrick Ollivander

### LINKED ISSUE #1238: 🚀 [Feature] Seed Data -  Garrick Ollivander
## 🚀 Feature

New Seed User `Garrick Ollivander` 

`Curious ... curious ...`
`Renowned wandmaker Mr Ollivander owns the wand shop Ollivanders: Makers of Fine Wands Since 382 BC in Diagon Alley. His shop is widely considered the best place to purchase a wand.`

### LINKED ISSUE #1258: 🚀 [Feature] Seed: User without Email confirmation
## 🚀 Feature

Seed: User without Email confirmation.

Could be done together with #1238

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
Garrick Ollivander has not confirmed his email yet.

### Issues
- fixes #1258
- fixes #1238

### CODE REVIEW

File: database/src/interface/UserInterface.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ export interface UserInterface {
   username?: string
   description?: string
   password?: BigInt
-  pubKey?: Buffer
-  privKey?: Buffer
+  pubKey?: Buffer | null
+  privKey?: Buffer | null
```
Comment (ulfgebhardt) [inline database/src/interface/UserInterface.ts:10]: The questionmak allows to leave the value - the definition of null should not be required

Comment (ulfgebhardt) [reply database/src/interface/UserInterface.ts:10]: Same above

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]

File: database/src/interface/UserInterface.ts (hunk 1 above)
Comment (Mogge) [reply database/src/interface/UserInterface.ts:10]: `should` is not `is`
I had to change this to get the typescript compiled

Comment (Mogge) [reply database/src/interface/UserInterface.ts:10]:
```
TSError: ⨯ Unable to compile TypeScript:
src/seeds/users/garrick-ollivander.seed.ts:7:31 - error TS2345: Argument of type '{ email: string; firstName: string; lastName: string; username: string; description: string; password: bigint; pubKey: null; privKey: null; emailHash: Buffer; createdAt: Date; emailChecked: boolean; ... 8 more ...; addBalance: boolean; }' is not assignable to parameter of type 'UserInterface'.
  Types of property 'pubKey' are incompatible.
    Type 'null' is not assignable to type 'Buffer | undefined'.
```

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**